src/Components/EditProfileModal/EditProfileModal.jsx

```diff
@@ -0,0 +1,102 @@
+import { useUserContext } from "Context";
+import { useState } from "react";
+import * as styles from "./EditProfileModal.module.css";
+
+export const EditProfileModal = ({
+  firstName,
+  bio,
+  profilePhoto,
+  coverPhoto,
+  setModal,
+}) => {
+  const { editProfile } = useUserContext();
+  const [editedData, setEditedData] = useState({
+    firstName: firstName,
```

Comment:
Instead of doing this, I think, you can use shorthand notation here. something like this,
```javascript
useState({
firstName,
bio,
profilePhoto,
coverPhoto,
})
```

As here the variable name is the same as the key na. So, I think this should also work.

Reply:
yes i will make the changes